processing file asmFiles/test.dual_llsc_debug.asm (1 of 1)
processing file asmFiles/test.dual_llsc_debug.asm (1 of 1)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/test.cfhazard_beq.asm (1 of 3)
--- memsim.hex	2024-11-12 18:21:08.900932819 -0500
+++ memcpu.hex	2024-11-12 18:21:23.523501658 -0500
@@ -1,240 +1,10 @@
-:04000000FFC06413C6
-:04000100FFC06113C8
-:04000200FFFF71B7D4
-:040003004031013354
-:04000400403404334D
-:04000500000071B7CF
-:040006000031013391
-:04000700003404338A
-:0400080040806613BB
-:0400090040402C0344
-:04000A0000106293ED
-:04000B00005C56B38C
-:04000C0000C064B319
-:04000D0000D0693383
-:04000E00210000EFDE
-:04000F0000106293E8
-:04001000005C52B38B
-:04001100405C06B396
-:040012000020631354
-:04001300006292B342
-:0400140040806613AF
-:040015000056063358
-:0400160000C069B30A
-:0400170000D06A3378
-:0400180000906633BB
-:04001900012066B3A9
-:04001A000130673317
-:04001B00014067B386
-:04001C00600062938B
-:04001D00FFC101130B
-:04001E00005120234A
-:04001F004000631327
-:040020000003238333
-:04002100FE038EE369
-:04002200208000EF4B
-:040023000041011384
-:04002400FFFFFFFFDC
-:04008000FFFFC1B706
-:04008100FFC0641345
-:04008200FFC0611347
-:0400830040310133D4
-:0400840040340433CD
-:04008500408066133E
-:0400860040402C03C7
-:040087000010629370
-:04008800005C56B30F
-:04008900002062935E
-:04008A00005692B3D7
-:04008B0000560633E2
-:04008C0040DC06B39B
-:04008D00014000EF3F
-:04008E004000629339
-:04008F0000106313E7
-:040090000062A02347
-:04009100FFFFFFFF6F
-:040092000040629335
-:0400930000206313D3
-:04009400006693333C
-:040095000062B23320
-:0400960002020A63F5
-:0400970000560FB34D
-:04009800000FAF03A3
-:0400990000CF8E63A3
-:04009A00FFCFA803E9
-:04009B00010F2233FC
-:04009C0000020863F3
-:04009D00010FA0238C
-:04009E00FFCF8F936E
-:04009F00FE9FF06F61
-:0400A00001EFA023A9
-:0400A1000042829304
-:0400A200FCDFF06F20
-:0400A3000000806772
-:0400A40000012283B2
-:0400A5000006966358
-:0400A6000007946358
-:0400A70007C0006F1F
-:0400A8000407806366
-:0400A90004068C635A
-:0400AA000006230326
-:0400AB0000072383A4
-:0400AC000073223388
-:0400AD0000020C63DE
-:0400AE000062A02329
-:0400AF0000428293F6
-:0400B00000460613ED
-:0400B100FFF686933D
-:0400B200FCDFF06F10
-:0400B3000072A02314
-:0400B40000428293F1
-:0400B50000470713E6
-:0400B600FFF7879336
-:0400B700FB9FF06F4C
-:0400B8000006230318
-:0400B9000062A0231E
-:0400BA0000428293EB
-:0400BB00FFF6869333
-:0400BC0000460613E1
-:0400BD000206826352
-:0400BE00FE9FF06F42
-:0400BF000007238390
-:0400C0000072A02307
-:0400C10000428293E4
-:0400C200FFF787932A
-:0400C30000470713D8
-:0400C400000784634A
-:0400C500FE9FF06F3B
-:0400C600000080674F
-:0401000000000001FA
-:0401010000000040BA
-:0401020000000001F8
-:0401030000000003F5
-:0401040000000004F3
-:0401050000000007EF
-:040106000000000BEA
-:040107000000000CE8
-:0401080000000015DE
-:0401090000000016DC
-:04010A0000000016DB
-:04010B0000000019D7
-:04010C000000001BD4
-:04010D000000001DD1
-:04010E0000000020CD
-:04010F0000000021CB
-:0401100000000029C2
-:0401110000000033B7
-:0401120000000036B3
-:0401130000000038B0
-:040114000000003DAA
-:040115000000003DA9
-:040116000000003EA7
-:0401170000000042A2
-:04011800000000479C
-:040119000000004B97
-:04011A000000004D94
-:04011B000000005090
-:04011C00000000518E
-:04011D00000000518D
-:04011E00000000538A
-:04011F000000005A82
-:040120000000006378
-:040121000000006476
-:0401220000000007D2
-:0401230000000007D1
-:0401240000000007D0
-:0401250000000008CE
-:0401260000000008CD
-:0401270000000008CC
-:0401280000000009CA
-:040129000000000AC8
-:04012A000000000FC2
-:04012B0000000010C0
-:04012C0000000011BE
-:04012D0000000017B7
-:04012E000000001AB3
-:04012F0000000020AC
-:0401300000000020AB
-:0401310000000023A7
-:0401320000000024A5
-:0401330000000026A2
-:0401340000000026A1
-:04013500000000289E
-:040136000000002F96
-:040137000000003094
-:040138000000003291
-:040139000000004181
-:04013A00000000427F
-:04013B00000000437D
-:04013C00000000447B
-:04013D000000005965
-:04013E000000005B62
-:04013F000000005C60
-:040140000000005C5F
-:040141000000006456
-:04018000000000017A
-:040181000000000377
-:040182000000000475
-:040183000000000771
-:040184000000000770
-:04018500000000076F
-:04018600000000076E
-:04018700000000086C
-:04018800000000086B
-:04018900000000086A
-:04018A000000000968
-:04018B000000000A66
-:04018C000000000B64
-:04018D000000000C62
-:04018E000000000F5E
-:04018F00000000105C
-:04019000000000115A
-:040191000000001555
-:040192000000001653
-:040193000000001652
-:040194000000001750
-:04019500000000194D
-:040196000000001A4B
-:040197000000001B49
-:040198000000001D46
-:040199000000002042
-:04019A000000002041
-:04019B000000002040
-:04019C00000000213E
-:04019D00000000233B
-:04019E000000002439
-:04019F000000002636
-:0401A0000000002635
-:0401A1000000002832
-:0401A2000000002930
-:0401A3000000002F29
-:0401A4000000003027
-:0401A5000000003224
-:0401A6000000003322
-:0401A700000000361E
-:0401A800000000381B
-:0401A9000000003D15
-:0401AA000000003D14
-:0401AB000000003E12
-:0401AC00000000410E
-:0401AD00000000420C
-:0401AE00000000420B
-:0401AF000000004309
-:0401B0000000004407
-:0401B1000000004703
-:0401B2000000004BFE
-:0401B3000000004DFB
-:0401B40000000050F7
-:0401B50000000051F5
-:0401B60000000051F4
-:0401B70000000053F1
-:0401B80000000059EA
-:0401B9000000005AE8
-:0401BA000000005BE6
-:0401BB000000005CE4
-:0401BC000000005CE3
-:0401BD0000000063DB
-:0401BE0000000064D9
-:0401BF0000000064D8
-:043FFE0000000600B9
+:040000000280051362
+:0400010000A00293C6
+:0400020001400313A3
+:040003000000066390
+:040004000065202350
+:04000500FFFFFFFFFB
+:04000600005520235E
+:04000700FFFFFFFFF9
+:04000A000000000AE8
 :00000001FF
processing file asmFiles/test.coherence1.asm (2 of 3)
processing file asmFiles/test.coherence2.asm (3 of 3)
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 1)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
